src/ops/arithmetic_test.ts

```diff
@@ -29,6 +29,26 @@ describeWithFlags('div', ALL_ENVS, () => {
     expectArraysClose(r, [1, 1, 1, 1, 2.5, 6 / 5]);
   });
 
+  it('integer division implements floor divide', () => {
+    const a = tf.tensor1d([-5, -4, 3, 2], 'int32');
+    const c = tf.tensor1d([3, 2, 2, 2], 'int32');
+
+    const r = tf.div(a, c);
+
+    expect(r.dtype).toEqual('int32');
+    expectArraysClose(r, [-2, -2, 1, 1]);
```

Comment:
I believe that this is actually rounding. 
Shouldn't the first value here, -2, be -1?  `Math.floor(-5/3)` should be -1.

Reply:
👍 
One more issue, if you do tf.div with 3/3, you get a 0:

```javascript
    const a = tf.scalar(3, 'int32');
    const c = tf.scalar(3, 'int32');

    const r = tf.div(a, c);

    expect(r.dtype).toEqual('int32');
    expectArraysClose(r, [1]);
```
results in an error:
	Error: Arrays differ: actual[0] = 0, expected[0] = 1.